txscript/standard_test.go

```diff
@@ -671,27 +698,52 @@ func TestMultiSigScript(t *testing.T) {
 	t.Parallel()
 
 	//  mainnet p2pk 13CG6SJ3yHUXo4Cr2RY4THLLJrNFuG3gUg
-	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKey(decodeHex("02192d7"+
-		"4d0cb94344c9569c2e77901573d8d7903c3ebec3a957724895dca52c6b4"),
+	pubkey1, err := chainec.Secp256k1.ParsePubKey(decodeHex("02192d7" +
+		"4d0cb94344c9569c2e77901573d8d7903c3ebec3a957724895dca52c6b4"))
+
+	if err != nil {
+		t.Errorf("Unable to parse pubkey (compressed): %v",
+			err)
+		return
+	}
+
+	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKeyCompressed(pubkey1,
 		&chaincfg.MainNetParams)
 	if err != nil {
 		t.Errorf("Unable to create pubkey address (compressed): %v",
 			err)
 		return
 	}
-	p2pkCompressed2Main, err := dcrutil.NewAddressSecpPubKey(decodeHex("03b0bd"+
```

Comment:
Please don't change this. There is no need since they are specifically being given the compressed bytes.